src/assets/styles/scss/base.scss

```diff
@@ -0,0 +1,90 @@
+/* Media Queries */
+
+@mixin respond($breakpoint) {
+  @if ($breakpoint == phone-narrow) {
+    @media (max-width: 25em) {
+      @content;
+    }
+  }
+
+  @if ($breakpoint == phone) {
+    @media (max-width: 37.5em) {
+      @content;
+    }
+  }
+
+  @if ($breakpoint == tab-port) {
+    @media (max-width: 56.25em) {
+      @content;
+    }
+  }
+
+  @if ($breakpoint == tab-land) {
+    @media (max-width: 75em) {
+      @content;
+    }
+  }
+
+  @if ($breakpoint == mission) {
+    @media (max-width: 99em) {
+      @content;
+    }
+  }
+
+  @if ($breakpoint == big-desktop) {
+    @media (min-width: 112.5em) {
+      @content;
+    }
+  }
+}
+
+/* Body Font */
+
+@mixin font-source-body(
+  $size: false,
+  $color: false,
+  $weight: false,
+  $lineHeight: false,
+  $letterSpace: false
+) {
+  font-family: SofiaProSoftW01-Regular;
+  @if ($size) {
+    font-size: $size;
+  }
+  @if ($color) {
+    color: $color;
+  }
+  @if ($weight) {
+    font-weight: $weight;
+  }
+  @if ($lineHeight) {
+    line-height: $lineHeight;
+  }
+  @if ($letterSpace) {
+    letter-spacing: $letterSpace;
+  }
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: #ffffff;
+}
+
+code {
```

Comment:
it's not obvious what this style does-- could we streamline this into the mixins? (like the font source example above)